src/spring/azext_spring/_params.py

```diff
@@ -268,6 +268,8 @@ def load_arguments(self, _):
                    help='A json file path for the persistent storages to be mounted to the app')
         c.argument('loaded_public_certificate_file', options_list=['--loaded-public-certificate-file', '-f'], type=str,
                    help='A json file path indicates the certificates which would be loaded to app')
+        c.argument('deployment_name', type=str,
+                   help='Name of the active deployment. Deployment named "default" will be created if this argument not provided', validator=validate_name)
```

Comment:
We should use the named argument `default="default"` to set the default value. With this we don't need to highlight it in the help text.

Check the `--sku` argument for reference. In the help output, it shows:

```
    --sku                                                   : Name of SKU. Enterprise is still in
                                                              Preview.  Allowed values: Basic,
                                                              Enterprise, Standard.  Default:
                                                              Standard.
```

Reply:
Good point! I will modify the help text as follows:
```
    --deployment-name                                           : Name of the active deployment.
                                                                  Default: default.
```